packages/MarkdownEditor-Tests.package/MarkdownInlineParserTest.class/instance/testFindOpenerForEmphasisDelimiter.st

```diff
@@ -13,18 +13,11 @@ testFindOpenerForEmphasisDelimiter
 		isRightFlanking: true; 
 		yourself.
 		
-	tokens := OrderedCollection new.
-	tokens add: opener.
-	tokens add: (MarkdownString new content: 'f'; yourself).
-	tokens add: closer.
-	inlineParser tokens: tokens.
+	inlineParser tokens 
+		addAll: {opener.
+				(MarkdownString new content: 'f'; yourself).
+				closer}.
 	
 	inlineParser findOpenerFor: closer 
 		ifFound: [:o | foundOpener := o].
-	self assert: opener equals: foundOpener.
-	
-	tokens remove: opener.
-	inlineParser tokens: tokens.
-	inlineParser findOpenerFor: closer 
-		ifFound: [:o | self fail]
-	
\ No newline at end of file
+	self assert: opener equals: foundOpener
```

Comment:
I suggested during development that you move the assert into the `ifFound:` and replace it with `self fail`.
Just interested: Why did you not?

Reply:
This didn't reach me... But yeah, it seams to be better.